_posts/2021-11-15-sr2022-kickstart-and-rules.md

```diff
@@ -0,0 +1,24 @@
+---
+title: SR2022 Game Announced
+---
+
+Student Robotics 2022 was Kickstarted on Saturday on our [livestream][livestream]. It was great to see the excitement bulding for our first in-person competition since SR2019.
+
+Our game this year, _This Way Up_, challenges teams to collect the most tin cans from the arena and ensure that they're up the right way for maximum points. When in a team's scoring zone, correctly oriented cans are worth 3 points, upside down cans are worth one point, and cans lying on their side are worth nothing. Tin cans on the arena floor are upside down at the start of each match, but those on the raised platform in the centre are already the right way up. Cans are marked with an inslating band along the bottom, so robots can detect the orientation by trying to pass an electric current through the can body.
+
+Teams are scored at the end of the match for collecting the most points. Full details, including the prizes available this year, are available in the [rulebook][rules].
+
+This year's teams have already been strategising their approach, and we can't wait to see their progress over the year!
+
+If you weren’t able to attend Kickstart this year, or would like a recap, you can:
+
+- watch our [livestream][livestream] where we explain the game and how this year's competition will work.
```

Comment:
This reads to me as though it's referring to an upcoming livestream rather than a past one.

Reply:
It would if the previous sentence wasn't past tense. It's also the same wording we've used in the past, so it's _probably fine_.

Reply:
You're right, that makes sense.